src/pytest_opentelemetry/instrumentation.py

```diff
@@ -49,10 +49,20 @@ def get_trace_parent(cls, config: Config) -> Optional[Context]:
 
         return None
 
+    @classmethod
+    def try_force_flush(cls) -> bool:
+        provider = trace.get_tracer_provider()
+
+        # Not all providers (e.g. ProxyTraceProvider) implement force flush
+        if hasattr(provider, 'force_flush'):
+            provider.force_flush()
+            return True
+        else:
+            return False
+
     def pytest_configure(self, config: Config) -> None:
         self.trace_parent = self.get_trace_parent(config)
 
-        # This can't be tested both ways in one process
```

Comment:
This comment was explaining the `#pragma: no cover` below, which I think is still true.  Maybe the comment should say that explicitly.

Reply:
This was an accidental deletion, apologies 😅 I think it is clear as is